openXCiOSFrameworkTests/NetworkDataTest.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  NetworkDataTest.swift
```

Comment:
As discussed, please delete the test classes for now. and once you are done with Unit Testing, you can include the correct classes